models.py

```diff
@@ -280,9 +281,10 @@ def __init__(
             if dropout > 0:
                 convs.append(torch.nn.Dropout(dropout))
             in_channels = out_channels
-
+            dilation = dilation if dilation else 1
+            h_out = torch.floor((h_out + 2*padding - dilation*(kernel - 1) - 1)/stride + 1)
```

Comment:
There is no option for `dilation` and we always pad by half the kernel size on each side (L270) so this can be probably be simplified to:

`h_out //= stride`